static/build/miyoo/app/lang/en.lang

```diff
@@ -130,5 +130,6 @@
 	"127":  "Default",
 	"128":  "Delete",
 	"129":  "Audio fix",
-	"130":  "Fixes"
+	"130":  "Fixes",
+	"131":  "NEXT"
```

Comment:
You can't just add new keys to `en.lang`, unless you've handled missing keys in `system/lang.h` ("131" is missing in all other languages! - Which means with the current implementation, it will be `NULL` if the language isn't set to english).